app/services/check_out_book.rb

```diff
@@ -0,0 +1,19 @@
+class CheckOutBook
+  def self.process(book:, user:)
+    new(book: book, user: user).process
+  end
+
+  def initialize(book:, user:)
+    @book = book
+    @user = user
+  end
+
+  def process
+    due_date = DateTime.now + Checkout::CHECKOUT_PERIOD_IN_DAYS.days
```

Comment:
I'd use `Time.current` here. The guideline I use is to use `Time` unless I *need* to use DateTime for something. I can't remember the last time I've needed DateTime though so Time is pretty much the go to class to use